tests/should_succeed/mathlibtest.jou

```diff
@@ -5,27 +5,57 @@ def main() -> int:
     printf("%d\n", abs(-1)) # Output: 1
     printf("%lld\n", llabs(-1145141919180L)) # Output: 1145141919180
     printf("%f\n", fabs(-114.514)) # Output: 114.514000
+    printf("%f\n", fma(1.0, 2.0, 3.0)) # Output: 5.000000
+
     printf("%f\n", ceil(3.14159265358979)) # Output: 4.000000
     printf("%f\n", floor(3.14159265358979)) # Output: 3.000000
+    printf("%f\n", fmod(1.14, 5.14)) # Output: 1.140000
+
     printf("%f\n", round(3.14159265358979)) # Output: 3.000000
-    printf("%f\n", acos(-1)) # Output: 3.141593
-    printf("%f\n", asin(-1)) # Output: -1.570796
-    printf("%f\n", atan(-1)) # Output: -0.785398
-    printf("%f\n", atan2(-1, -1)) # Output: -2.356194
+    printf("%lld\n", lround(3.1415926)) # Output: 3
+    printf("%f\n", remainder(1.14, 5.14)) # Output: 1.140000
+
     printf("%f\n", cos(1.04719753333333)) # Output: 0.500000
     printf("%f\n", sin(0.785398149999999)) # Output: 0.707107
     printf("%f\n", tan(2.45)) # Output: -0.828017
-    printf("%f\n", fmod(9.3, 3.1)) # Output: 0.000000
-    printf("%f\n", sqrt(5)) # Output: 2.236068
-    printf("%f\n", cbrt(27)) # Output: 3.000000
-    printf("%f\n", log(114)) # Output: 4.736198
-    printf("%f\n", log10(114)) # Output: 2.056905
-    printf("%f\n", log2(114)) # Output: 6.832890
-    printf("%f\n", exp(3)) # Output: 20.085537
-    printf("%f\n", exp2(3)) # Output: 8.000000
-    printf("%f\n", pow(2, 2)) # Output: 4.000000
-    printf("%f\n", ldexp(10, 2)) # Output: 40.000000
+    printf("%f\n", acos(-1.0)) # Output: 3.141593
+    printf("%f\n", asin(-1.0)) # Output: -1.570796
+    printf("%f\n", atan(-1.0)) # Output: -0.785398
+    printf("%f\n", atan2(-1.0, -1.0)) # Output: -2.356194
+
+    printf("%f\n", copysign(4.0, -1.0)) # Output: -4.000000
+    printf("%f\n", nextafter(0.0, -1.0)) # Output: -0.000000
+
+    printf("%f\n", cosh(3.0)) # Output: 10.067662
+    printf("%f\n", sinh(4.0)) # Output: 27.289917
+    printf("%f\n", tanh(45.0)) # Output: 1.000000
```

Comment:
I don't like the 45 in these tests, because it looks confusingly like an angle, but that's not what it is. For example, the `tanh()` function calculates `(exp(x)-exp(-x)) / (exp(x)+exp(-x))`, where `exp(45)` is really large and `exp(-45)` is really small; therefore the division is close to 1. This has nothing to do with 45-degree angles.